presentation/planner/src/main/java/com/together/study/planner/main/PlannerViewModel.kt

```
@@ -103,6 +136,180 @@ internal class PlannerViewModel @Inject constructor(
+                )
+            }
+
+            state.copy(plannerSubjectState = UiState.Success(updatedSubjects))
+        }
+
+        checkJobs[taskId]?.cancel()
+
+        checkJobs[taskId] = viewModelScope.launch {
+            try {
+                updateTaskCheckedUseCase(taskId, newValue)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)